### DISCUSSION TITLE
Can't install socat utility on Mac

### DISCUSSION THREAD

**Discussion by abdulgamer786** [Homebrew/discussions#1276] [Getting started] [answered]
when I run `brew install socat` I get this:
`==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "socat".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.`

Is there a new name for `socat`?. How do I solve this problem. Thank you in advance!

**Comment by carlocab** [comment]
What's the output of `brew config` and `brew doctor`?

**Comment by abdulgamer786** [comment]
`brew config` output 
`HOMEBREW_VERSION: 3.0.1-80-gfa638d2
ORIGIN: https://github.com/Homebrew/brew
HEAD: fa638d29d89f611aa5eabb01d7eef8785bffd051
Last commit: 8 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: (none)
Core tap last commit: never
Core tap branch: HEAD
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: N/A
Git: 2.15.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.2.3-x86_64
CLT: N/A
Xcode: N/A
` 



`brew doctor` output:

`Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/muhsinhamzah/opt/anaconda3/bin/icu-config
  /Users/muhsinhamzah/opt/anaconda3/bin/krb5-config
  /Users/muhsinhamzah/opt/anaconda3/bin/freetype-config
  /Users/muhsinhamzah/opt/anaconda3/bin/xslt-config
  /Users/muhsinhamzah/opt/anaconda3/bin/libpng16-config
  /Users/muhsinhamzah/opt/anaconda3/bin/libpng-config
  /Users/muhsinhamzah/opt/anaconda3/bin/xml2-config
  /Users/muhsinhamzah/opt/anaconda3/bin/python3-config
  /Users/muhsinhamzah/opt/anaconda3/bin/curl-config
  /Users/muhsinhamzah/opt/anaconda3/bin/ncursesw6-config
  /Users/muhsinhamzah/opt/anaconda3/bin/pcre-config
  /Users/muhsinhamzah/opt/anaconda3/bin/python3.8-config

Warning: No developer tools installed.
Install the Command Line Tools:
  xcode-select --install


Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout 

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /usr/local/bin/com.docker.cli
  /usr/local/bin/docker
  /usr/local/bin/docker-compose
  /usr/local/bin/docker-credential-desktop
  /usr/local/bin/docker-credential-ecr-login
  /usr/local/bin/docker-credential-osxkeychain
  /usr/local/bin/hub-tool
  /usr/local/bin/hyperkit
  /usr/local/bin/kubectl
  /usr/local/bin/kubectl.docker
  /usr/local/bin/notary
  /usr/local/bin/vpnkit
  /usr/local/lib/docker/cli-plugins
`

**Comment by carlocab** [comment, marked as answer]
Please run
```
xcode-select --install
rm -rf /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core
brew tap homebrew/core
brew update
```

**Reply by abdulgamer786** [reply]
Ok I am doing it thank you

**Comment by abdulgamer786** [comment]
ok thank you everything works!
